libs/package.json

```diff
@@ -4,7 +4,7 @@
   "audius": {
     "releaseSHA": "d3c913701122ef44083733b36b12ce4c3668ee3e"
   },
-  "description": "",
+  "description": "Audius SDK",
```

Comment:
Made this nice change to force trigger the flows, but felt like I should just keep it here because why not